application-services/custom/influxdb-export/main.go

```diff
@@ -0,0 +1,62 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+//
+// Copyright (C) 2020 IOTech Systems LTD
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package main
+
+import (
+	"fmt"
+	"os"
+
+	influxTransforms "app-service-influx/pkg/transforms"
+
+	"github.com/edgexfoundry/app-functions-sdk-go/appsdk"
+	sdkTransforms "github.com/edgexfoundry/app-functions-sdk-go/pkg/transforms"
+)
+
+func main() {
+
+	// 1) First thing to do is to create an instance of the EdgeX SDK, giving it a service key
+	edgexSdk := &appsdk.AppFunctionsSDK{
+		ServiceKey: "InfluxDBExport", // Key used by Registry (Aka Consul)
+	}
+
+	// 2) Next, we need to initialize the SDK
+	if err := edgexSdk.Initialize(); err != nil {
+		message := fmt.Sprintf("SDK initialization failed: %v\n", err)
+		edgexSdk.LoggingClient.Error(message)
+		os.Exit(-1)
+	}
+
+	// 3) Initialize the MQTT addressable/configuration
+	mqttConfig := sdkTransforms.MqttConfig{}
+	addressable, err := influxTransforms.LoadMQTTAddressable(edgexSdk)
+
+	if err != nil {
+		edgexSdk.LoggingClient.Error(fmt.Sprintf("SDK MQTT Addressable initialize failed: %v\n", err))
+		os.Exit(-1)
+	}
+	mqttSender := sdkTransforms.NewMQTTSender(edgexSdk.LoggingClient, *addressable, nil, mqttConfig, false)
```

Comment:
`NewMQTTSender` has been deprecated. 
Please use `NewMQTTSecretSender` with Authmode='none' . Then you will not need to create addressable